content/docs/concepts/authorization/v1.x/_index.md

```diff
@@ -29,7 +29,7 @@ The following diagram shows a high-level overview of Container Storage Modules f
 **NOTE:** PowerScale OneFS implements its own form of Role-Based Access Control (RBAC). Authorization does not enforce any role-based restrictions for PowerScale. To configure RBAC for PowerScale, refer to the PowerScale OneFS [documentation](https://www.dell.com/support/home/en-us/product-support/product/isilon-onefs/docs).
 
 ## Authorization Components Support Matrix
-Authorization consists of 2 components - The authorization sidecar, bundled with the driver, communicates with the Authorization proxy server to validate access to Storage platforms. The authorization sidecar is backward compatible with older Authorization proxy server versions. However, it is highly recommended to have the Authorization proxy server and sidecar installed from the same release of Container Storage Modules.
+Authorization consists of 2 components - The authorization sidecar, bundled with the driver, communicates with the Authorization proxy server to validate access to Storage platforms. We highly recommend that customers install or upgrade the Authorization proxy server and sidecar from the same release of CSM. This practice helps ensure optimal performance and compatibility, avoiding potential issues from version discrepancies.
```

Comment:
I cannot tell what the two components are. After "Authorization consists of two components", you should give the two components that you are speaking about. After you introduce what the two components are, you can then write your description of how the components work together.

Reply:
Fixed